modules/rosapps/applications/screensavers/blankscr/scrnsave.rc

```diff
@@ -59,3 +59,6 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_blankscr.ico"
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
 #endif
+#ifdef LANGUAGE_IT_IT
```

Comment:
These entries should be placed in alphabetical order (same remark for the other modified .rc files).

Reply:
I did not understand it, I got used to Transifex and I'm no longer used to translating directly from code.